frappe/__init__.py

```diff
@@ -281,10 +281,11 @@ def init(site: str, sites_path: str = ".", new_site: bool = False, force: bool =
 	local.qb.get_query = get_query
 	setup_redis_cache_connection()
 
-	if not _qb_patched.get(local.conf.db_type):
+	if not _one_time_setup.get(local.conf.db_type):
 		patch_query_execute()
 		patch_query_aggregation()
 		_register_fault_handler()
```

Comment:
ummm, why don't we do just do this statically? idk reasons for doing this on first request only :thinking:

Reply:
Oh now I get it, this is required post worker forking :smile: